nipype/interfaces/fsl/utils.py

```diff
@@ -1700,7 +1700,28 @@ def _gen_filename(self, name):
             return self._list_outputs()['out_file']
         return None
 
+      
+class FslOrientInputSpec(FSLCommandInputSpec):
+    in_file = File(exists=True, desc='input file', argstr='%s', position=2, mandatory=True)
```

Comment:
I would suggest adding `copyfile=True`, since this tool modifies the input file. This will only have an effect in workflows/nodes. (See https://nipype.readthedocs.io/en/latest/devel/interface_specs.html)

```suggestion
    in_file = File(exists=True,
                   copyfile=True,
                   desc='input file',
                   argstr='%s',
                   position=2,
                   mandatory=True)
```